wiki/Music_theory/Beat/en.md

```diff
@@ -1,5 +1,20 @@
+---
+tags:
+  - beats
+  - white line
+  - white lines
+  - white tick
+  - white ticks
+  - whole beat
+  - whole beats
+---
+
 # Beat
 
-A **beat** is a repeating unit of time in music, in particular the pulse of the music to which the listener taps to. In osu!, you can compare them to the white ticks in the [editor timeline](/wiki/Client/Beatmap_editor/Timelines) which form the backbone of the rhythm of the song.
+A **beat** is a repeating event or unit of time in music, which creates the rhythm that a person would tap or nod their head to while listening.
+
+![In-game screenshot of editor timeline](img/timeline.png "Each white tick represents a beat in the music")
+
+A whole beat is referred to as 1/1 and is contained between two adjacent white lines on the [timeline](/wiki/Client/Beatmap_editor/Timelines). Equal amounts of beats are grouped into [measures](/wiki/Music_theory/Measure), which start from [downbeats](/wiki/Music_theory/Downbeat) represented by big white ticks.
 
-![In-game screenshot of editor timeline](img/timeline.png "All white ticks represent a beat in the music")
+Other notes are expressed through a beat by dividing or multiplying its length. The beatmap editor's [beat snap divisor](/wiki/Client/Beatmap_editor/Beat_Snap_Divisor) does that automatically for a mapper, allowing for changing the temporal granularity on the fly.
```

Comment:
i'm sure this won't cause any confusion

```suggestion
Other notes are expressed through a beat by dividing or multiplying its length. The beatmap editor's [beat snap divisor](/wiki/Client/Beatmap_editor/Beat_Snap_Divisor) does that automatically for a mapper, allowing for changing the temporal granularity on the fly.

Since a beat snap divisor of 1/1 corresponds to a whole beat, one beat is also commonly referred to as a "1/1", a half of a beat (an eighth note) is called a "1/2", a quarter of a beat (a sixteenth note) is called a "1/4", etc.
```

Reply:
reads like just what I was looking